src/problem_depot/problems/affine.jl

```diff
@@ -280,6 +280,24 @@ end
         @test p.optval ≈ 147 atol = atol rtol = rtol
     end
 
+    # *Scalar* indexing creates a column vector
```

Comment:
This test is probably misplaced since it doesn’t use a solver

Reply:
I will move